src/const/config.js

```diff
@@ -0,0 +1,11 @@
+// import { rest } from 'src/utils/api';
+import axios from 'axios';
+
+const request = axios.create(); // todo axios instance need to be singleton
+request.interceptors.response.use((res) => res.data);
```

Comment:
hmm not sure about this. What if we need something outside the `data` field of an api response?

Reply:
we have `meta` for example